tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -137,8 +135,7 @@ def test_smoke(self):
                 password2=password
             )
             self.assertIsNotNone(self.user.search(user_name))
-            is_admin_role_selected = self.user.admin_role_to_user(user_name)
-            self.assertTrue(is_admin_role_selected)
+            self.assertTrue(self.user.user_admin_role_toggle(user_name))
```

Comment:
We have recent improvement for `admin_role_to_user` method. We changed its name and that bring a regression